src/components/AssetsTable/AssetsTable.test.jsx

```diff
@@ -81,9 +81,21 @@ const numberOfImages = defaultProps.assetsList.filter(asset => asset.thumbnail).
 const numberOfStudioButtons = defaultProps.assetsList.filter(asset => asset.portable_url).length;
 const numberOfWebButtons = defaultProps.assetsList.filter(asset => asset.external_url).length;
 
+let parentElement;
+let querySelector;
 let wrapper;
 
 describe('<AssetsTable />', () => {
+  beforeEach(() => {
+    querySelector = global.document.querySelector;
+    parentElement = global.document.createElement('div');
```

Comment:
What does `global` do? I thought that `document.createElement('div')` would work.

Reply:
I thought global was necessary, but I removed it and it still works, so I'll do that instead.